src/action/HistoryImpl.ts

```diff
@@ -35,7 +35,7 @@ class HistoryImpl implements History {
     window.addEventListener("keydown", (event) => {
       if (
         (!this.redoStack.length && !this.undoStack.length) ||
-        shouldRejectKeystrokes()
+        shouldRejectKeystrokes('ArrowDown')
```

Comment:
Hm, why is `ArrowDown` significant here?

Reply:
I added this here just for consistency. On line 35 we subscribe to keydown event which is ArrowDown in tinykeys lib.

Reply:
Here is my mistake